single-pool/program/src/instruction.rs

```diff
@@ -43,6 +43,19 @@ pub enum SinglePoolInstruction {
     ///  12. `[]` Stake program
     InitializePool,
 
+    ///   Restake the pool stake account if its validator is force-deactivated and later
+    ///   recreated with the same vote account.
+    ///
+    ///   0. `[]` Validator vote account
+    ///   1. `[]` Pool account
+    ///   2. `[w]` Pool stake account
+    ///   3. `[]` Pool stake authority
+    ///   4. `[]` Clock sysvar
+    ///   5. `[]` Stake history sysvar
+    ///   6. `[]` Stake config sysvar
+    ///   7. `[]` Stake program
+    ReactivatePool,
```

Comment:
I got a little tripped up on the name, but I see what you use it, since `redelegate` means something else. The only thing I was able to come up with is `DelegatePoolStake` which would say literally what's happening, but wouldn't be as clear as to when it would be useful. If you prefer `ReactivatePool`, I'm good with it

Reply:
`ReactivateStake`? `ReactivatePoolStake`?

Reply:
last one seems fine